static/style.css

```diff
@@ -1,36 +1,194 @@
 }
 
-#title {
-    position: absolute;
-    top: 50%;
-    left: 0;
-    right: 0;
-    color: #fff;
+.navbar {
+  display: flex;
+  min-width: 100%;
+  height: minmax(100px, 8vh);
+  justify-content: flex-start;
+  align-items: center;
+  background-color: #0000004d;
+}
+
+.navbar a {
+  z-index: 100;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 8vh;
+  width: 100px;
+  color: #f2f2f2;
+  text-decoration: none;
+  font-size: 20px;
+  transition: all 0.5s ease;
+}
+
+.navbar a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+.navbar a[data-active="true"] {
+  background-color: #4caf50;
+  color: white;
+}
+
+.h1 {
+  background-color: #ffffff00;
+  color: white;
+  font: 60px arial;
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+.h2 {
+  background-color: #ffffff00;
+  color: #90ee90;
+  font: 20px arial;
+  text-align: center;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  height: 82vh;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+@media screen and (max-width: 700px) {
+  .navbar {
     text-align: center;
-    font-family: Lato;
-    font-weight: 300;
-    font-size: 50px;
-    letter-spacing: 10px;
-    margin-top: -60px;
-    padding-left: 10px;
+  }
+}
+
+.social-menu ul {
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+.social-menu ul li {
+  list-style: none;
+  margin: 0 10px;
+}
+.social-menu ul li .fa {
+  color: #000000;
+  font-size: 25px;
+  line-height: 50px;
+  transition: 0.5s;
+}
+.social-menu ul li .fa:hover {
+  color: #ffffff;
+}
+.social-menu ul li a {
+  position: relative;
+  display: block;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  background-color: white;
+  text-align: center;
+  transition: 0.5s;
+  box-shadow: 0px 7px 5px rgba(0, 0, 0, 0.5);
+}
+
+.info {
+  z-index: 100;
+  height: 82vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  color: #fff;
+  margin: 0 100px;
+  font-size: 20px;
+}
+
+@media screen and (max-width: 800px) {
+  .info {
+    font-size: 18px;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .info {
+    font-size: 16px;
+  }
+}
+
+@media screen and (max-width: 500px) {
+  .info {
+    margin: 0 10px;
+  }
+  .info p {
+    padding: 10px 20px;
+    width: 20ch;
+  }
+}
+
+.info p {
+  font-weight: 300;
+  background: rgba(0, 0, 0, 0.75);
+  font-family: 'Lato', sans-serif;
+  align-items: center;
+  padding: 10px 50px;
+  width: 50ch;
+}
+
+.social-menu ul li a:hover {
+  transform: rotate(0deg) skew(0deg) translate(0, -10px);
+}
+.social-menu ul li:nth-child(1) a:hover {
+  background-color: #431c53;
+}
+.social-menu ul li:nth-child(2) a:hover {
+  background-color: #55acee;
+}
+.social-menu ul li:nth-child(3) a:hover {
+  background-color: #e4405f;
+}
+
+.footer {
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  bottom: 0;
+  padding: 10px 10px;
+  width: 100vw;
+  background-color: rgba(0, 0, 0, 0.75);
```

Comment:
Appearing 2 different misplaced backgrounds on the index.html

Reply:
Fixed This